src/libs/Network/MainQueue.js

```diff
@@ -84,7 +80,7 @@ function process() {
  * Non-cancellable requests like Log would not be cleared
  */
 function clear() {
-    networkRequestQueue = _.filter(networkRequestQueue, (request) => !request.data.canCancel);
+    networkRequestQueue = [];
```

Comment:
@neil-marcellini we can keep the log requests here so we still don't need the canCancel